### ISSUE TITLE
Hanja in figures 22 and 24

### ISSUE THREAD

**Issue by r12a** [w3c/klreq#3]
Reporter: Norbert Lindenberg
http://www.w3.org/TR/2013/WD-klreq-20130514

4.1.2: Are arrangements like figures 22 and 24 still used today? The high percentage of hanja seems to indicate that they are older samples.

**Comment by peremen** [comment]
Yes, both figures (newspaper scraps) are quite old. Since late 1990s no newspapers in South Korea are using vertical writing, and that is probably the reason why this document contains old examples. Technically it is still possible, but practically it is used in very limited number of publications.
